packages/app/src/app/pages/Profile/Showcase/index.js

```diff
@@ -18,8 +18,8 @@ class Showcase extends React.Component {
 
   render() {
     const { store } = this.props;
-    const sandbox = store.profile.showcasedSandbox;
-    const isCurrentUser = store.profile.isProfileCurrentUser;
+    const sandbox = store.profile.showcasedSandbox.get();
```

Comment:
So if I understand correctly these `.get()` calls will have the right typings returned in the future? What's the reason of using `get`?

Reply:
Yeah, you can read about how fluent is used in an app here https://github.com/cerebral/cerebral/blob/canary/docs/guides/typescript.md

There are currently two reason for using computed.

1. To compute sorting, filtering and other kinds of performance affected code. Computed caches the value

2. Only computed has access to parent state

Ideally we would just use a `getter` on this situation, but because we need access to parent state we need to use a computed. I am not sure how to solve `getters` with parent state in a good way. Like technically we can solve everything, the challenge is the typing :)

This is still under evaluation though, so maybe we find a way and can refactor this in the next round :)